Source/RakNetSocket2_Berkley.cpp

```diff
@@ -8,7 +8,7 @@
  *
  */
 
-#include "EmptyHeader.h"
+#include "EmptyHeader.h"

```

Comment:
^ What's the difference here? Is it only the new line character? I would prefer to keep style or formatting changes to a minimum, as they might confuse the reader when they look at this commit in the future.